wikipedia-dark.user.css

```diff
@@ -645,6 +645,176 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
   input[type="checkbox"] {
     background-color: var(--base-color);
   }
+  input#searchButton[type="submit"] {
+    border-left-color: var(--gray-a) !important;
+    filter: invert(1) !important;
+    background-color: var(--cm-10) !important;
+  }
+  .mw-parser-output > #Vorlage_Alternative,
+  .infobox td[style*="border:1px #aaa solid;"],
+  .infobox th[style*="background:#d0d0d0;"],
+  .mw-parser-output > table.vertical-navbox,
+  .tux-message-item-compact,
+  .nomobile.plainlist,
+  .infobox td[style^="background-color:#cfea9e;"],
+  tr[style^="border-top:2px solid #aaaaaa"],
+  div[style^="z-index"][style*="background:transparent;"][style*="1px solid #CCC"],
+  .infobox[style*="border: solid 1px silver"],
+  tr[style*="border:1px solid #aaa;"] {
+    border-color: var(--gray-5) !important;
+  }
+  .frb-inline-main,
+  .mw-parser-output .wwikipedii td,
+  .mw-parser-output .wwikipedii th,
+  .infobox.geography .mergedbottomrow td,
+  .infobox.geography .mergedbottomrow th,
+  .mw-parser-output .sidebar-collapse .sidebar-above,
+  .mw-parser-output .sidebar-collapse .sidebar-below,
+  .tux-editor-insert-buttons button,
+  .infobox > tbody > tr:not([style]) > th[style*="background: #ccf"],
+  .ve-ui-cxDesktopContext .ve-ui-cxLinkContextItem-sourceBody {
+    border-color: var(--gray-5);
+  }
+  .uls-settings-trigger {
+    border-color: var(--gray-2) !important;
+  }
+  .wikibase-toolbar-container.wikibase-toolbar-container {
+    background: transparent !important;
+  }
+  .mw-parser-output .maf-info,
+  .mw-parser-output .wwikipedii tr {
+    background: var(--gray-18);
```

Comment:
Just a nit really, but with a few exceptions, `background` should be replaced with `background-color` as a more specific property to modify color without affecting other `background-*` properties that may be in place. We do the ssme thing for borders.

Exceptions to this would be when you intentionally want to overwrite the full background property (e.g. replace `background-image`).

Reply:
Yeah, I know about it but I noticed very many `background` rules in style, where background-color should be enough. All in all, it's better not to duplicate potential mistakes . Thx.